README.md

```diff
@@ -64,50 +71,52 @@ learn more about topics not directly related to a language to get this right! If
 I was doing Le Wagon today, I would read the resources below directly after it.
 
 1. Learn the most valuable Computer Science concepts in 1 afternoon with
-   [Computer Science Distilled](https://leanpub.com/computer-science-distilled).
-   Best book ever!
-2. Improve your Git level by reading
-   [Ry's Git Tutorial](https://www.amazon.com/Rys-Git-Tutorial-Ryan-Hodson-ebook/dp/B00QFIA5OC),
-   it is free and amazing!
-3. Enjoy a funny dive into DevOps with
-   [The Phoenix Project](https://www.amazon.com/Phoenix-Project-DevOps-Helping-Business/dp/0988262592)
+   [Computer Science Distilled]. Best book ever!
+1. Improve your Git level by reading
+   [Ry's Git Tutorial], it is free and amazing!
+1. Enjoy a funny dive into DevOps with [The Phoenix Project].
 
 ### Improve your Ruby skills
 
 If you know Ruby well, Rails will be so much easier!
 
-1. Read [Eloquent Ruby](http://eloquentruby.com/)
-2. Read [99 bottles of OOP](https://www.sandimetz.com/99bottles/)
+1. Read [Eloquent Ruby]
+1. Read [99 bottles of OOP]
+1. Do you Remember the "old days" when you were running Rubocop tests in your
+   terminal ? Do you run any still ? I'm sure you do. But let's [automate] it !
 
 #### To go further
 
-1. Subscribe to [Ruby Tapas](https://www.rubytapas.com/)
+1. Subscribe to [Ruby Tapas]
 
 ### Master Rails
 
 Because of course!
 
-1. Read the [Rails guides](http://guides.rubyonrails.org/)
-2. Read [The Rails 5 Way](https://leanpub.com/tr5w)
+1. Read the [Rails guides]
+1. Read [The Rails 5 Way]
 
 #### To go further
 
-1. Read
-   [Rails Anti-Patterns](https://www.amazon.com/Rails-AntiPatterns-Refactoring-Addison-Wesley-Professional/dp/0321604814)
-2. Subscribe to [Go Rails](https://gorails.com)
+1. Read [Rails Anti-Patterns]
+1. Subscribe to [Go Rails]
 
 ### Get fluent in Javascript
 
 Javascript is primordial but you don't know it well after Le Wagon. Take some
-time to improve your skills.
+   time to improve your skills. [Wes Bos] :crown: is one of the best teacher to
+   start learning JavaScript. He have a good amount of free stuff. And some
+   other paid ones.
 
-1. Do the free [Javascript30](https://javascript30.com/) web course
-2. Learn the last Javascript syntax with [ES6 for everyone](https://es6.io/)
+1. You fall asleep during LeWagon's morning classes ? Want to know the crazy new
```

Comment:
I don't really get the reason of this text here? ;)

Reply:
Eh ok. 👅